src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -531,7 +529,7 @@ function systemStatusGetUpdateStatus() {
 		},
 		dataType: 'html',
 		success: function(data){
-			$('#system-information-widget-updatestatus').html(data);
+			$('[id^=widget-system_information] #system-information-widget-updatestatus').html(data);
```

Comment:
This change makes it update all copies of 'system-information-widget-updatestatus' that might be in multiple copies of the widget.